docs/changelog.md

```diff
@@ -3,6 +3,9 @@ id: changelog
 title: Changelog
 sidebar_label: Changelog
 ---
+## v2.3.0
+
+- Added foreach clause [#351](https://github.com/memgraph/memgraph/pull/351)
```

Comment:
We also have a note about not supporting `FOREACH` in [`cypher-manual/differences-in-cypher-implementations.md`](https://github.com/memgraph/docs/blob/68e7e7a87ec3d25ed19e510a84b2c87515e104f0/cypher-manual/differences-in-cypher-implementations.md?plain=1#L26-L27)

Reply:
Really good catch